src/main/kotlin/org/rust/ide/refactoring/implementMembers/impl.kt

```diff
@@ -104,9 +107,32 @@ private fun insertNewTraitMembers(
             val whitespaces = createExtraWhitespacesAroundFunction(addedMember, next)
             existingMembers.addAfter(whitespaces, addedMember)
         }
+
+        if (selectedElement == null)
+        {
```

Comment:
Could you, please, format curly braces (in all changed code) according to [style guide](https://kotlinlang.org/docs/reference/coding-conventions.html#formatting)?